packages/drivers/odsp-driver/src/odspCache.ts

```diff
@@ -120,10 +120,12 @@ export interface IPersistedCache {
     updateUsage(entry: ICacheEntry, seqNumber: number): void;
 
     /**
-     * Removes all the entries from the cache for given docId.
-     * @param docId - DocId for which all entries needs to be deleted from cache.
+     * Removes the entries from the cache for given parametres.
+     * @param file - file entry to be deleted.
+     * @param type - Type of entry to be deleted for a given file.
```

Comment:
If we keep type/key here, can you please put a comment that they are optional, and if they are not supplied, it means implementer of this interface has to delete all records that match remaining (supplied) info

Reply:
No, I will remove these.